scripts/stage-release.sh

```diff
@@ -169,7 +162,17 @@ then
     exit
 fi
 
+# create GitHub release via CLI https://cli.github.com/manual/gh_release_create
+gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md" --draft
+echo "Draft release created. This must be manually published."
+
+ORIGINAL_GH_DEFAULT_REPO=$(gh repo set-default -v | (grep -v 'no default repository' || true))
+gh repo set-default $(git remote get-url ${REPOSITORY})
```

Comment:
Do we need all of this if we're running via actions? There should only be the one upstream when run as an action, so we should be able to set-and-forget this somewhere at the top of this script (or even as part of the workflow setup).

Reply:
My intention was to keep scripts runnable from command line (for debugging purposes).